### DISCUSSION TITLE
Crash on textViewportLayoutController.viewportRange unwrapping

### DISCUSSION THREAD

**Discussion by Mx-Iris** [krzyzanowskim/STTextView#32] [Questions]
<img width="1832" alt="iShot_2023-10-02_02 35 00" src="https://github.com/krzyzanowskim/STTextView/assets/61279231/2ebd6c3c-87f9-4a06-abc1-b5a06e2cc696">

**Comment by krzyzanowskim** [comment]
thanks. Anything in particular to trigger that?

**Reply by Mx-Iris** [reply]
If the heightTracksTextView of textContainer is enabled, there is a probability of triggering

**Comment by krzyzanowskim** [comment]
This is addressed (I believe) since 0.9.0
